### PR TITLE
Fix issue #91 DEBUG in common.py dealt as str after loaded from config file

### PR DESCRIPTION
**Pull request by oplier**
Hello Charles,

I nearly read and tried the entire project but only found this tiny issue :)

Br,
oplier

### CODE REVIEW

File: git_cc/common.py [hunk 1]
```diff
@@ -226,6 +226,7 @@ def path(path, args='-m'):
 cfg.read()
 CC_DIR = path(cfg.get(CFG_CC))
 DEBUG = cfg.getCore('debug', True)
+DEBUG = DEBUG == str(True)
```
Comment (charleso) [inline git_cc/common.py]: Hey @oplier , thanks for the PR. Two nits. 

1. You mind making it a single line
2. My python is terrible, but would it be safer to do `str(cfg.getCore('debug', True)) == str(True)`?

**Comment by charleso** [comment]
@oplier Sorry, I hope I didn't push you away? The PR is appreciated.

**Comment by oplier** [comment]
You are so sweet :)

Just now, in order to reduce commit times, I used "git reset --hard" to drawback previous commit.

(I am not so familiar with Github so I suppose Github does not allow modification on existing pull request.)

Previous commit was through Github web page, I intend to commit again use git command. But I am in my work place under firewall. So it took me quite a while by using SSH terminal of Google Could to commit and push again.

**Comment by charleso** [comment]
@oplier If you did a `git reset --hard` and then force pushed you would have reset the branch back to `master`, which closed the PR. You would have needed a new/extra commit to push first. Just a guess though.

**Comment by oplier** [comment]
@charleso OK, I see. I will be careful next time. Yesterday, I submitted another PR #93, with the newest changes.

**Pull request closed without merging**